posts/2022-10-14-ledger.markdown

```diff
@@ -0,0 +1,66 @@
+# High level summary
+
+The ledger team is still primarily focused on addressing
+[technical](https://github.com/input-output-hk/cardano-ledger/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3Acredit_card%3A+technical-debt%22)
+[debt](https://github.com/input-output-hk/cardano-ledger/projects/3).
+We now have the infrastructure for versioning our serialization schemes,
+which we continue to put into action.
+We have made first steps towards getting proper support for the
+[formal ledger repository](https://github.com/input-output-hk/formal-ledger-specifications)
+(in particular, we've added nix builds and
+continuous integration support).
+We are wrapping up an investigation of the performance of a critical
+function used by the consensus layer for leader checks.
+Finally, we are improving the packaging and versioning of our code.
+
+# Lower level summary
+
+## Completed Technical Debt
+
+- Because the Shelley ledger era was a complete re-write of the Byron ledger era, a lot of
+  our code lives in the `cardano-ledger-shelley` package, though with hindsight we can say that
+  much of it should live in `cardano-ledger-core`.
+  We continue to move things to `cardano-ledger-core`, and have much more to come. [[pull-3059]]
+- We now have the infrastructure to support **versioned** serialization schemes.
+  The inability to do this has caused us caused us a lot difficulties,
```

Comment:
```suggestion
  The inability to do this has caused us a lot difficulties,
```

Reply:
it's even done permanent damage to my brain my brain.